configs/a64fx.yaml

```diff
@@ -41,8 +38,15 @@ Branch-Predictor:
   Global-History-Length: 11
   RAS-entries: 8
   Fallback-Static-Predictor: "Always-Taken"
-LSQ-L1-Interface:
-  Access-Latency: 5
+Memory-Hierarchy:
+  Cache-Line-Width: 256
+  L1-Data:
+    Access-Latency: 5
+    Size: 65536 # 64 KiB
+  DRAM:
+    Access-Latency: 200
```

Comment:
Can we not put the access latency of DRAM to 5 for the time being as that's the only thing being used currently? Since there  are caches we can just put a comment in each .yaml file stating values not in use yet

Reply:
Done